pkg/aws/ec2.go

```diff
@@ -21,6 +21,15 @@ import (
 
 const MultiPlatformManaged = "MultiPlatformManaged"
 
+var defaultInstanceTags = []types.Tag{
+	{Key: aws.String("Project"), Value: aws.String("Konflux")},
+	{Key: aws.String("Owner"), Value: aws.String("konflux-infra@redhat.com")},
+	{Key: aws.String("ManagedBy"), Value: aws.String("Konflux Infra Team")},
+	{Key: aws.String("app-code"), Value: aws.String("ASSH-001")},
+	{Key: aws.String("service-phase"), Value: aws.String("Production")},
```

Comment:
Good question. If they're non-static, then no doubts they're must be configurable (fully or partially - i'd like to avoid "dead" - never changing config options)